src/model/Note.ts

```diff
@@ -797,11 +797,12 @@ export class Note {
         if (points != null && points.length > 0 && this.bendType === BendType.Custom) {
             let isContinuedBend: boolean = this.isTieDestination && this.tieOrigin!.hasBend;
             this.isContinuedBend = isContinuedBend;
-            if (points.length === 4) {
+            let isFourPointsBend = (points.length === 4);
+            if (points.length === 4 || points.length === 3) {
                 let origin: BendPoint = points[0];
                 let middle1: BendPoint = points[1];
-                let middle2: BendPoint = points[2];
-                let destination: BendPoint = points[3];
+                let middle2: BendPoint =  isFourPointsBend ? points[2] : points[1];
+                let destination: BendPoint = points.at(-1)!; //returns last element
```

Comment:
`.at()` is currently not supported on all platforms. Can you either rever this to classical operators or extend the C# & Kotlin platforms accoringly?

Reply:
reverted